contrib/jupyter/build.py

```diff
@@ -7,6 +7,9 @@
 import os, sys
 import subprocess
 
+def install_external_packages():
+    subprocess.check_call([sys.executable, "-m", "pip", "install", "protobuf"])
```

Comment:
I'm not sure if that's the best way to do it. `protobuf` is a dependency of the `orbitutils` package, right?

If yes, I think it should go into the `install_requires` property of `setup.py`.

Reply:
Ah, yes, that sounds better!